frontend/src/pages/WorkspaceTeam/components/InviteMemberModal.tsx

```diff
@@ -223,6 +242,42 @@ function InviteMemberModal({
 													? options.join(', ')
 													: options
 											}}
+											value={
+												allProjects
+													?.filter((p) =>
+														newProjectIds.includes(
+															p?.id ?? '',
+														),
+													)
+													?.map((p) => p?.name)
+													?.filter(Boolean) as
+													| string[]
+													| undefined
+											}
+											onValueChange={(
+												values: {
+													name: string
+												}[],
+											) =>
+												setNewProjectIds(
+													(allProjects
+														?.filter((p) =>
+															values
+																.map(
+																	(v) =>
+																		v.name,
+																)
+																.includes(
+																	p?.name ??
+																		'',
+																),
+														)
+														?.map((p) => p?.id)
+														?.filter(Boolean) as
+														| string[]
+														| undefined) ?? [],
+												)
```

Comment:
Makes sense, but seems like a lot of extra logic for something we should be able to grab directly from the form state on submission 🤔 - wondering if we could actually remove this `newProjectIds` and `setNewProjectIds` entirely and just use the state stored in the form rather than React state.

Don't worry about refactoring right now to get this bug fixed, but something to consider at some point.

Reply:
good call - cleaned this up!